filters/auth/tokeninfo.go

```diff
@@ -28,6 +28,15 @@ type TokeninfoOptions struct {
 	Timeout      time.Duration
 	MaxIdleConns int
 	Tracer       opentracing.Tracer
+
+	// CacheSize configures the size of the tokeninfo cache.
+	// The cache evicts least recently used items first.
+	CacheSize int
```

Comment:
docstring does not tell if this is number of items or Bytes or ..

Reply:
Fixed